config.w32

```diff
@@ -115,11 +122,15 @@ if (PHP_MONGODB != "no") {
     MONGOC_HAVE_SASL_CLIENT_DONE: 0,
     MONGOC_HAVE_WEAK_SYMBOLS: 0,
     MONGOC_NO_AUTOMATIC_GLOBALS: 1,
+    MONGOC_CC: "",
+    MONGOC_USER_SET_CFLAGS: "",
+    MONGOC_USER_SET_LDFLAGS: ""
   };
 
   if (CHECK_LIB("ssleay32.lib", "mongodb", PHP_MONGODB) &&
       CHECK_LIB("libeay32.lib", "mongodb", PHP_MONGODB) &&
-      CHECK_HEADER_ADD_INCLUDE("openssl/ssl.h", "CFLAGS_MONGOC")) {
+      CHECK_LIB("crypt32.lib", "mongodb", PHP_MONGODB) &&
```

Comment:
This is necessary for both OpenSSL (currently used) and Secure Channel (will be used in the future).